android/app/src/main/res/values/color.xml

```diff
@@ -12,6 +12,7 @@
     <color name="grey">#E0E0E0</color>
     <color name="white">#FFFFFF</color>
     <color name="black">#000000</color>
+    <color name="track_header_default">#546E7A</color>
```

Comment:
Why are you setting a default color? If a track has no color associated with it then don't display any color.